core/scripts/Voting.js

```diff
@@ -190,15 +192,22 @@ class VotingSystem {
 
   async runBatchCommit(requests, roundId) {
     const commitments = [];
+    const skipped = [];
+    const failures = [];
 
     for (const request of requests) {
       // Skip commits if a message already exists for this request.
       // This does not check the existence of an actual commit.
       if (await this.getMessage(request, roundId)) {
+        skipped.push(request);
         continue;
       }
 
-      commitments.push(await this.constructCommitment(request, roundId));
+      try {
+        commitments.push(await this.constructCommitment(request, roundId));
```

Comment:
One error case I think we should consider for a follow-up is when the commit or reveal txns fail.

Reply:
I agree, we should notify on that error case too. I'll add that in a follow up.